src/components/visualizations/chart/highcharts/plugins/autohideLabels/autohideColumnLabels.ts

```diff
@@ -125,19 +162,36 @@ function toggleStackedLabels() {
     }
 }
 
+function toggleStackedLabels() {
+    const { yAxis } = this;
+    const { length: axisNumber } = yAxis;
+
+    // CL-10676 - Return if yAxis is undefined
+    if (!yAxis || axisNumber === 0) {
```

Comment:
Just use this here after !yAxis is enough IMHO
`if (!yAxis || yAxis.length === 0) {`